CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
I would rather merge this PR than git cherry-pick your changes and create a new one. CRLF I know is what caused the reformat, but it makes that section harder to review. I still think it's worth breaking it into another PR. Can you cherry-pick that commit to a different branch and then remove it from this branch? Then we can merge since I think you have resolved all of the other comments.